flink-core/src/main/java/org/apache/flink/api/common/state/AppendingState.java

```diff
@@ -58,10 +58,11 @@ public interface AppendingState<IN, OUT> extends State {
      * of values. The next time {@link #get()} is called (for the same state partition) the returned
      * state will represent the updated list.
      *
-     * <p>If null is passed in, the state value will remain unchanged.
+     * <p>If null is passed in, the behaviour is undefined (implementation related).
```

Comment:
Could we unify the behavious of all sub classes?
I think "undefined" maybe also confuse users.

Reply:
The current situation is that all implementations are not unified. This comment describe the current situation.

Reply:
Sure. I could get this.
How about adding TODO description here? Then users could also get the reason.
Then let's unify them in 2.0.